src/hash_table.js

```diff
@@ -0,0 +1,133 @@
+class Node {
+  constructor(key,value) {
+    this.key = key
+    this.value = value
+    this.next = null
+  }
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null
+  }
+
+  addNode(key,value) {
+    if(!this.head) {
+      this.head = new Node(key, value)
+    }
+    else {
+      let currentNode = this.head
```

Comment:
A linked list has constant insertion time( O(1) ), but it looks like you are inserting at the tail of the list, instead of the head. This changes the insertion time complexity to O(n) which is incorrect. Can you change the function to insert the new node at the front of the list?